addons/binding/org.openhab.binding.energenie/ESH-INF/config/bindingConfig.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="http://eclipse.org/smarthome/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/config-description/v1.0.0
+        http://eclipse.org/smarthome/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="energenie:binding:config">
+		<parameter name="user" type="text">
+			<label>User's email address</label>
+			<required>true</required>
+			<context>email</context>
+		</parameter>
+		<parameter name="password" type="text">
+			<label>User's password</label>

```

Comment:
Yes and no. It supports api keys, but the user is not able to see his api key until he makes an authenticated request to the Rest API using the ... password. So I would not like to force people doing this.